boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -26,10 +26,19 @@ qemu: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 	$(call check_defined, OPENTITAN_BOOT_ROM)
 	$(QEMU) -M opentitan -kernel $^ -bios $(OPENTITAN_BOOT_ROM) -nographic -serial mon:stdio
 
+
+qemu-gdb: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
+	$(call check_defined, OPENTITAN_BOOT_ROM)
+	$(QEMU) -gdb tcp:localhost:9000 -S -M opentitan -kernel $^ -bios $(OPENTITAN_BOOT_ROM) -nographic -serial mon:stdio
```

Comment:
Instead of `-gdb tcp:localhost:9000` can we use `-s` which is just shorthand for `-gdb tcp:localhost:1234`?

That way we use the default QEMU ports

Reply:
Good idea!